frontend/src/pages/projects/screens/spawner/storage/AddExistingStorageField.tsx

```diff
@@ -35,7 +35,7 @@ const AddExistingStorageField: React.FC<AddExistingStorageFieldProps> = ({
     () =>
       loaded
         ? storages.reduce((acc: TypeaheadSelectOption[], pvc) => {
-            if (!existingStorageNames?.includes(pvc.metadata.name)) {
+            if (!existingStorageNames?.includes(getDisplayNameFromK8sResource(pvc))) {
```

Comment:
why are you using display name here now. shouldn't it still be metadata name if existingStorageNames did not change

Reply:
when `display-name` and `metadata.name` are different incase of this example,existingStorageNames stores `display-name` not the `metadata.name`  so we need this condition here
```
`kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: pvc-test-name
  namespace: dsp-wb-test
  labels:
    opendatahub.io/dashboard: 'true'
  annotations:
    openshift.io/description: ''
    openshift.io/display-name: pvc-test-display-name
  finalizers:
    - kubernetes.io/pvc-protection
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 10Gi
  storageClassName: standard-csi
  volumeMode: Filesystem
status:
  phase: Pending 
  `
```